### PR TITLE
Feature/sc 5676 refactoring page objects admin login

### PR DESCRIPTION
**Pull request by Grzesiek80**
# Description

https://ticketsystem.schul-cloud.org/browse/SC-5676

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by kgorner** [comment]
PR (scenario @adminCreateClassWithoutParticipants)
is blocked by bug: https://ticketsystem.schul-cloud.org/browse/SC-5883

File: page-objects/pages/DashboardPage.js [hunk 1]
```diff
@@ -1 +1,53 @@
-/*[url/dashboard]*/
\ No newline at end of file
+/*[url/dashboard]*/
+'use strict';
+
+const { expect } = require('chai');
```
Comment (kgorner) [inline page-objects/pages/DashboardPage.js:4]: what is chai?

Comment (Grzesiek80) [reply page-objects/pages/DashboardPage.js:4]: chai is a test framework

Comment (Grzesiek80) [reply page-objects/pages/DashboardPage.js]: we don't need "chai" import there, we have access to this globally (without any imports). It was removed

**Pull request merged**